File: java/09_strings.java
public class Strings {
    public static void main(String[] args) {
        String name = "ulas";
        System.out.println(name.toUpperCase());
        System.out.println(name.toLowerCase());
        System.out.println(name.length());

        int age = 20;
        System.out.println(name + " is " + age + " years old");

        System.out.println(name.substring(0, 2)); // ul

        System.out.println(name.contains("ula"));
        System.out.println(name.startsWith("u"));
    }
}

File: rust/09_strings.rs
fn main() {
    let name = "ulas";
    println!("{}", name.to_uppercase());
    println!("{}", name.to_lowercase());
    println!("{}", name.len());

    let age = 20;
    println!("{} is {} years old", name, age);

    println!("{}", &name[0..2]); // ul

    println!("{}", name.contains("ula"));
    println!("{}", name.starts_with('u'));
}
